core/test/fixtures/fraggle-rock/reports/sample-flow-result.json

```diff
@@ -8766,7 +8766,7 @@
                 },
                 {
                   "origin": "https://mnl4bjjsnz-dsn.algolia.net",
-                  "serverResponseTime": 0
+                  "serverResponseTime": 263.2025
                 }
```

Comment:
seems cool

Reply:
This came from these estimates:
<img width="243" alt="image" src="https://github.com/GoogleChrome/lighthouse/assets/4071474/1b9da9bb-2c04-4c4c-9a76-f26cc0d705a0">

in `_estimateResponseTimeByOrigin`'s `Math.max(ttfb - rtt, 0)`, `ttfb - rtt` is somehow negative so we get a zero as an estimate. It's questionable if we should keep that as an estimate...

Reply:
The zero estimate came from this record: `https://mnl4bjjsnz-dsn.algolia.net/1/indexes/dev_OFFICE_SCENES/query`, `ttfb 49` (slightly more than the rtt estimate of 49.56)

The second, higher estimate came from: `https://mnl4bjjsnz-dsn.algolia.net/1/indexes/dev_OFFICE_SCENES/query`,  `ttfb 575`

it's reasonable for query time to be so variable for such a website, so I think taking the average-ish value here (via the median changes in this PR) is good.